### ISSUE TITLE
Can't create a new git branch

### ISSUE THREAD

**Issue by ynwd** [Microsoft/vscode#68679]
- VSCode Version: 1.31.1 (1.31.1)
- OS Version: MacOS Mojave 10.14.3 (18D109)

**Comment by kieferrm** [comment]
Please provide the concrete steps to reproduce the problem. Thanks!

**Comment by raldred** [comment]
I think this could be because the behaviour of the "create new branch" option has changed recently.
After entering a branch name and pressing enter the user is now presented with a second list to select select the ref in which to base the branch from. On quick glance it's easy to see the confusion.

![Creating a new branc](https://media.giphy.com/media/23g6M8Roi4cY5A7qp5/giphy.gif)

**Comment by Markus-bt** [comment]
Took me quite a while and several branches, created on command line to find the solution to my confusion here.
As a quick help for users, I would suggest, to change the first entry "HEAD" to something which better indicates the second use of this menue like "DERIVE FROM... HEAD

**Comment by joaomoreno** [comment]
The quick open mentions "Select a ref to create a new branch from".

I'll close this and see whether more users hit this confusion. It might just be a small "getting used to it" bump.

**Comment by raldred** [comment]
@joaomoreno I believe the requirement to select the ref is an unnecessary step for the majority of users. I think VScode should by default support the most common path first.

I'm pretty sure I'm not alone in saying that I have NEVER found myself wanting to create a branch from anything else other than HEAD

**Comment by Markus-bt** [comment]
I didn't even recognize that quickref... 
I did not expect anything to stand there except stuff I already knew so I wasn't looking (User Expectancy)
We do several branches that depend on other branches, but we usually opened the correct branch and then created a new one. So, getting used to it.
Closing is OK, but there are at least two other issues with this theme, I think.
..and we shouldn't keep writing comments in a closed issue, so that's all from me.
